pdfs/app/Main.hs

```diff
@@ -0,0 +1,20 @@
+
+import Data.ByteString as LBS
+import Data.Text as Text
+
+import Pdf.Scanner
+
+main :: IO ()
+main = do
+    [pdf, json, out] <- getArgs
```

Comment:
I think it's worth at least printing a help message if a wrong number of arguments is supplied.